packages/frontend2/src/server/features/data-availability/get-da-risk-entries.ts

```diff
@@ -1,14 +1,30 @@
 import { daLayers } from '@l2beat/config'
 import { toDaBridge } from './utils/get-da-bridge'
+import { getDaEconomicSecurity } from './utils/get-da-economic-security'
+import { getUniqueProjectsInUse } from './utils/get-da-projects'
+import {
+  getDaProjectsTvl,
+  pickTvlForProjects,
+} from './utils/get-da-projects-tvl'
 import { getDaRisks } from './utils/get-da-risks'
 
 export async function getDaRiskEntries() {
+  const economicSecurity = await getDaEconomicSecurity()
+  const uniqueProjectsInUse = getUniqueProjectsInUse()
+  const tvlPerProject = await getDaProjectsTvl(uniqueProjectsInUse)
+  const getSumFor = pickTvlForProjects(tvlPerProject)
+
   return daLayers.flatMap((daLayer) =>
     daLayer.bridges.map((bridge) => ({
       slug: daLayer.display.slug,
       daLayer: daLayer.display.name,
       daBridge: toDaBridge(bridge),
-      risks: getDaRisks(daLayer, bridge),
+      risks: getDaRisks(
+        daLayer,
+        bridge,
+        getSumFor(bridge.usedIn),
```

Comment:
I would extract this to a variable

Reply:
https://github.com/l2beat/l2beat/pull/4504/commits/5f6f821ee6079b9b29cb05aafee6e7a71c8394ac